### PR TITLE
glog static lib flags fix

### PR DESCRIPTION
**Pull request by jiayuehua**
without those fix, CI report error:
https://github.com/vesoft-inc/nebula/runs/5417305671?check_suite_focus=true
image

### CODE REVIEW

File: project/externals/glog.cmake [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Copyright (c) 2019 vesoft inc. All rights reserved.
 #
-# This source code is licensed under Apache 2.0 License.
+# This source code is licensed under Apache 2.0 License,
+# attached with Common Clause Condition 1.0, found in the LICENSES directory.
```
[the remark this answers is not in the document]
Comment (jiayuehua) [reply project/externals/glog.cmake:4]: ok

**Review by yixinglu — APPROVED** [review]

**Comment by dutor** [comment]
Could you please elaborate the error and reason? Maybe this is not the right way to address this issue.

> ERROR: something wrong with flag 'timestamp_in_logfile_name' in file '/build/build/glog/source/src/logging.cc'.  One possibility: file '/build/build/glog/source/src/logging.cc' is being linked both statically and dynamically into this executable.

**Comment by jiayuehua** [comment]
> Could you please elaborate the error and reason? Maybe this is not the right way to address this issue.
> 
> > ERROR: something wrong with flag 'timestamp_in_logfile_name' in file '/build/build/glog/source/src/logging.cc'.  One possibility: file '/build/build/glog/source/src/logging.cc' is being linked both statically and dynamically into this executable.

error CI report error:
[vesoft-inc/nebula/runs/5417305671?check_suite_focus=true](https://github.com/vesoft-inc/nebula/runs/5417305671?check_suite_focus=true)

with this fix, program can run properly on my local machine.

https://stackoverflow.com/questions/30475415/caffe-compilation-error-gflags-cc-is-being-linked-both-statically-and-dynamica  
I've worked out a way to debug this issue analytically. In my case, I was cross-compiling for an older ABI, so apt-get wasn't an option and I was compiling all dependencies manually.

First let's take a look at what this issue actually is. In the Google GFlags library, flags are declared through global objects. When the global object's constructor is run, it calls into the GFlags library to register that command line flag. If the global constructor gets run multiple times (due to multiple versions of the library containing it being loaded into memory), then the GFlags register method dies with an error.

What does GLog have to do with this? Well, GLog uses GFlags, and it has globally declared flag objects. Even if GFlags is linked correctly, if the GLog library gets loaded multiple times, you get an error pointing to logging.cc in GLog.

**Comment by dutor** [comment]
> > Could you please elaborate the error and reason? Maybe this is not the right way to address this issue.
> > > ERROR: something wrong with flag 'timestamp_in_logfile_name' in file '/build/build/glog/source/src/logging.cc'.  One possibility: file '/build/build/glog/source/src/logging.cc' is being linked both statically and dynamically into this executable.
> 
> error CI report error: [vesoft-inc/nebula/runs/5417305671?check_suite_focus=true](https://github.com/vesoft-inc/nebula/runs/5417305671?check_suite_focus=true)
> 
> with this fix, program can run properly on my local machine.
> 
> https://stackoverflow.com/questions/30475415/caffe-compilation-error-gflags-cc-is-being-linked-both-statically-and-dynamica I've worked out a way to debug this issue analytically. In my case, I was cross-compiling for an older ABI, so apt-get wasn't an option and I was compiling all dependencies manually.
> 
> First let's take a look at what this issue actually is. In the Google GFlags library, flags are declared through global objects. When the global object's constructor is run, it calls into the GFlags library to register that command line flag. If the global constructor gets run multiple times (due to multiple versions of the library containing it being loaded into memory), then the GFlags register method dies with an error.
> 
> What does GLog have to do with this? Well, GLog uses GFlags, and it has globally declared flag objects. Even if GFlags is linked correctly, if the GLog library gets loaded multiple times, you get an error pointing to logging.cc in GLog.

So the right way is to prevent glog from getting linked multiple times, other than to fix the way to build glog.

**Pull request closed without merging**